numpy/lib/_stride_tricks_impl.py

```diff
@@ -546,13 +546,12 @@ def broadcast_arrays(*args, subok=False):
     # return np.nditer(args, flags=['multi_index', 'zerosize_ok'],
     #                  order='C').itviews
 
-    args = tuple(np.array(_m, copy=None, subok=subok) for _m in args)
+    args = [np.array(_m, copy=None, subok=subok) for _m in args]
```

Comment:
Does this really help? I thought that these days list comprehension and creating a tuple via an iterator made very little speed difference, and `*args` should be very slightly faster for a tuple.

Reply:
It does help, although you are right the cost of list comprehensions has gone down. On Python 3.12:
```
In [16]: import dis
    ...: args=[1, 2, 3]
    ...: %timeit tuple(2*j for j in args)
    ...: %timeit tuple([2*j for j  in args])
298 ns ± 22.1 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
108 ns ± 0.781 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```
So about 200ns for a small size, which does matter in the benchmarks.

Reply:
Yes, I confirm this. It is funny because I really thought python had solved the speed difference, but clearly I was wrong.